src/main/resources/application.yml

```diff
@@ -152,7 +152,7 @@ spring:
     # database-platform: org.hibernate.dialect.PostgreSQLDialect
 
     generate-ddl: false
-    hibernate.ddl-auto: none
```

Comment:
Undo this please. The database schema is created via the Liquibase. The only instruction needed is to drop and recreate the database as defaults are what we require.

Reply:
Change reverted